integration-tests/tests/pages/search.page.ts

```diff
@@ -27,9 +27,12 @@ export class SearchPage {
     }
 
     async enableSearchFields(...fieldLabels: string[]) {
-        await this.page.getByRole('button', { name: 'Add Search Fields' }).click();
+        const addButton = this.page.getByRole('button', { name: 'Add Search Fields' });
+        // wait for button to be hydrated and visible
+        await expect(addButton).toBeVisible({ timeout: 10000 });
+        await addButton.click();
```

Comment:
Ah I made this change while debugging the integration tests it is not required
```suggestion
        await this.page.getByRole('button', { name: 'Add Search Fields' }).click();
```